x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/cases/cell.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo } from 'react';
+import { EuiLink, EuiSkeletonText } from '@elastic/eui';
+import { Tooltip as CaseTooltip } from '@kbn/cases-components';
+import type { CaseTooltipContentProps } from '@kbn/cases-components';
+import { ALERT_CASE_IDS } from '@kbn/rule-data-utils';
+import { CellComponentProps } from '../types';
+import { useCaseViewNavigation } from './use_case_view_navigation';
+import { Case } from '../hooks/api';
+
+const formatCase = (theCase: Case): CaseTooltipContentProps => ({
+  title: theCase.title,
+  description: theCase.description,
+  createdAt: theCase.created_at,
+  createdBy: {
+    username: theCase.created_by.username ?? undefined,
+    fullName: theCase.created_by.full_name ?? undefined,
+  },
+  status: theCase.status,
+  totalComments: theCase.totalComment,
+});
+
+const CasesCellComponent: React.FC<CellComponentProps> = ({ isLoading, alert, cases }) => {
+  const { navigateToCaseView } = useCaseViewNavigation();
+
+  const caseIds = alert[ALERT_CASE_IDS] ?? [];
+
+  const validCases = caseIds
+    .map((id) => cases.get(id))
+    .filter((theCase): theCase is Case => theCase != null);
+
+  if (validCases.length === 0) {
+    return null;
```

Comment:
I've seen that if there is no linked case, the cell is just empty. I guess it's happening because of this return and I'm wondering if we should render '--' or even call the DefaultRenderer to keep consistency with other empty cell values

See difference between empty values of other cells compared to case

<img width="962" alt="Screenshot" src="https://user-images.githubusercontent.com/17549662/220145273-fe32e082-234e-46d2-8c32-0aad65f78118.png">

Reply:
You are right. I will call the `v
`